extras/cloudsql-multicluster/README.md

```diff
@@ -114,22 +140,36 @@ kubectx cluster2
 kubectl apply  -n ${NAMESPACE} -f ../cloudsql/kubernetes-manifests
 ```
 
-11. **Run the Multi-cluster Ingress setup script.** This registers both GKE clusters to Anthos with "memberships," and sets cluster 1 as the "config cluster" to administer the Multi-cluster Ingress resources.
+12. **Delete the LoadBalancer type frontend services**. With `MultiClusterIngress`
+    we will be using a `MultiClusterService`.
+
+```sh
+kubectx cluster1
+kubectl delete svc frontend -n ${NAMESPACE}
+
+kubectx cluster2
+kubectl delete svc frontend -n ${NAMESPACE}
+```
+
+13.  **Run the Multi-cluster Ingress setup script.** This registers both GKE clusters to Anthos with ***"memberships"*** and sets cluster 1 as the ***"config cluster"*** to administer the Multi-cluster Ingress resources.
 
 ```
 ./register_clusters.sh
 ```
 
 
-12. **Create Multi-cluster Ingress resources for global routing.**  This YAML file contains two resources a headless Multicluster Kubernetes Service ("MCS") mapped to the `frontend` Pods, and a multi cluster Ingress resource, `frontend-global-ingress`, with `frontend-mcs` as the MCS backend. Note that we're only deploying this to Cluster 1, which we've designated as the multicluster ingress "config cluster."
+14. **Create Multi-cluster Ingress resources for global routing.**  This YAML file contains two resources a headless Multicluster Kubernetes Service ("MCS") mapped to the `frontend` Pods, and a multi cluster Ingress resource, `frontend-global-ingress`, with `frontend-mcs` as the MCS backend. Note that we're only deploying this to Cluster 1, which we've designated as the multicluster ingress "config cluster."
 
 ```
 kubectx cluster1
 kubectl apply -n ${NAMESPACE} -f multicluster-ingress.yaml
 ```
 
 
-13. **Verify that the multicluster ingress resource was created.** Look for the `Status` field to be populated with two Network Endpoint Groups (NEGs) corresponding to the regions where your 2 GKE clusters are running. This may take a few minutes.
+15. **Verify that the multicluster ingress resource was created.** Look for the `Status` field to be populated with two Network Endpoint Groups (NEGs) corresponding to the regions where your 2 GKE clusters are running.
```

Comment:
Nitpick (feel free to ignore): This doc uses "Multi-cluster Ingress", "multicluster ingress", "Multi Cluster Ingress", and some other variations. It would be cool if this doc consistently used [Multi Cluster Ingress](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-ingress).

Reply:
Fixed in 8002f3a4546fd205951fd05b94e1ddc64eff33c0